lib/eslint.js

```diff
@@ -527,7 +527,7 @@ function getRuleReplacementMessage(ruleId) {
     if (ruleId in replacements.rules) {
         const newRules = replacements.rules[ruleId];
 
-        return "Rule \'" + ruleId + "\' was removed and replaced by: " + newRules.join(", ");
+        return `Rule \'${ruleId}\' was removed and replaced by: ${newRules.join(", ")}`;
```

Comment:
Can these escaped single-quotes be non-escaped now? Actually, I'm not sure why they were escaped to begin with